include/api/CDataFrameTrainBoostedTreeClassifierRunner.h

```diff
@@ -29,12 +29,6 @@ class API_EXPORT CDataFrameTrainBoostedTreeClassifierRunner final
     using TReadPredictionFunc = std::function<TDouble2Vec(const TRowRef&)>;
     using TReadClassScoresFunc = std::function<TDouble2Vec(const TRowRef&)>;
 
-    enum EPredictionFieldType {
-        E_PredictionFieldTypeString,
-        E_PredictionFieldTypeInt,
-        E_PredictionFieldTypeBool
-    };
```

Comment:
This can stay here if we don't bother to pass to inference model writer directly.

Reply:
👍